droidlet/dashboard/web/src/StateManager.js

```diff
@@ -230,6 +230,16 @@ class StateManager {
     }
   }
 
+  /**
+   * sends commands to backend for mobile button navigation
+   * similar to keyHandler, but keyHandler is for web and arrow keys
+   */
+  buttonHandler(commands) {
+    if (commands.length > 0) {
+      this.socket.emit("command", commands);
```

Comment:
Can you use a more descriptive name for the socket event? eg. `update_image_settings`

Reply:
I could do this, however I copied the socket.emit logic from the keyhandler function (in this same file). If I were to rename this socket event, I would also have to update the logic above, and then change the backend to listen for <new name> instead of "command." It's by no means a problem, I just want to be 100% on what to change (since the backend and the function above is already existing code)